### ISSUE TITLE
Do not require end date for events (or allow content editors to select zero)

### ISSUE THREAD

**Issue by smayden** [az-digital/az_quickstart#725]
Currently, content editors must enter an end date/time when creating a new event. Many events to do not specify an end time, so ideally this would be optional.

On possible solution I heard in a session on Smart Date at Drupal Con was to create a custom time increment of "zero" that content editors could select.

**Comment by smayden** [comment]
I attended a session at Drupal Con on Smart Date where they recommended creating a duration of "Zero" to handle events with not end time. This is also mentioned in the Smart Date documentation: https://www.drupal.org/project/smart_date

I'm able to set this up in QS 2, but when I select the zero duration the event shows as "All Day" and does not show the specified start time. 

Not sure if this is a Smart Date bug (didn't see in the issue queue) or some sort of conflict with our configuration. I tried switching off of the AP Style date formatter, but that made no difference.

![image](https://user-images.githubusercontent.com/10873398/118407077-d8b10800-b633-11eb-9474-74bc8442b1d5.png)

![image](https://user-images.githubusercontent.com/10873398/118407087-e5356080-b633-11eb-9714-371c7daceb74.png)
